packages/http-client/src/http-client.ts

```diff
@@ -300,6 +325,46 @@ function logRequestInformation(request: HttpRequestConfig) {
   }
 }
 
+/**
+ * Builds a [[DestinationHttpRequestConfig]] for the given destination, merges it into the given requestConfig
+ * and executes it (using Axios).
+ * One overloading, that accepts [[HttpRequestConfigWithOrigin]] as a parameter, is deprecated and replaced the function [[executeHttpRequestWithOrigin]].
+ * @param destination - A destination or a destination name and a JWT.
+ * @param requestConfig - Any object representing an HTTP request.
+ * @param options - An [[HttpRequestOptions]] of the http request for configuring e.g., csrf token delegation. By default, the SDK will not fetch the csrf token.
+ * @returns A promise resolving to an [[HttpResponse]].
+ */
+export function executeHttpRequest<T extends HttpRequestConfig>(
+  destination: DestinationOrFetchOptions,
+  requestConfig: T,
+  options?: HttpRequestOptions
+): Promise<HttpResponse>;
+/**
+ * @deprecated This overloading is replaced by the function [[executeHttpRequestWithOrigin]].
```

Comment:
[req] Incorrect word, I think.
```suggestion
 * @deprecated This overload is replaced by the function [[executeHttpRequestWithOrigin]].
```